sdk/storage/AzureStorageBlob/Source/Options/ClientMethodOptions.swift

```diff
@@ -38,6 +38,9 @@ public struct ListContainersOptions: AzureOptions {
     /// A client-generated, opaque value with 1KB character limit that is recorded in analytics logs.
     public let clientRequestId: String?
 
+    /// A token used to make a best-effort attempt at cancelling a request.
```

Comment:
Soooo... based on the above, this should be "canceling" 🤦

Reply:
Fixed.